x-pack/plugin/runtime-fields/src/main/java/org/elasticsearch/xpack/runtimefields/mapper/AbstractScriptFieldType.java

```diff
@@ -206,6 +206,9 @@ protected final void doXContentBody(XContentBuilder builder, boolean includeDefa
         toXContent.accept(builder, includeDefaults);
     }
 
+    // Placeholder Script for source-only fields
+    private static final Script DEFAULT_SCRIPT = new Script("");
```

Comment:
Oh this is tricky! Could we use `null` here instead? I'm not a big of making a `Script` that no one can compile. It'll have stuff like `engine=painless` which is isn't.

Reply:
On second though, I think `null` is difficult here because we dereference the script all over the place. I plan to rework a lot of those dereferences eventually anyway so maybe we can just swear that we'll remove this "very soon" and be ok with it.

Reply:
let's add a TODO just to make it clear that this is temporary?